infra/base-images/base-runner/coverage_helper

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python3
```

Comment:
this file should have .py extension.

our most code is python2, should we use python3 for this ?

Reply:
Changed the extension.

Runner image doesn't have Python2 installed. I'm not sure if it's worth adding another dependency just for one script.